src/components/uxtabarea/uxtabarea.js

```diff
@@ -0,0 +1,21 @@
+Ractive.extend({
+	template: RactiveF.templates.uxtabarea,
+	oninit: function () {
+
+		var tabLinks = this.findComponent('uxtablinks');
+		var tabPanes = this.findComponent('uxtabpanes');
+
+		var tabLink = tabLinks.findAllChildComponents('uxtablink');
+		var tabPane = tabPanes.findAllChildComponents('uxtabpane');
+
+		_.each(tabLink, function (link, i) {
+			var childPane;
+			if (childPane = tabPane[i]) {
```

Comment:
How did this get pass "boss" option??

http://jshint.com/docs/options/#boss

Created issue #37 to track adding jshint to gulp build.